feature_layers/feature-collection-layer/src/main/java/com/esri/samples/feature_collection_layer/FeatureCollectionLayerSample.java

```diff
@@ -67,8 +68,14 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
-      // create amp and set it to be displayed in this view
-      ArcGISMap map = new ArcGISMap(Basemap.createOceans());
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a map with a basemap style
```

Comment:
```suggestion
      // create a map with the oceans basemap style
```

Reply:
For reference, we are going with option 2: `create a map with the insert-type basemap style`